src/lib/index.ts

```diff
@@ -22,3 +27,21 @@ export const IconContext = createContext<IconContextProps>({
   weight: "regular",
   mirrored: false,
 });
+
+export const renderPathForWeigth = (
```

Comment:
Was planning on wrapping all icons in `React.memo` to prevent unnecessary calls to the render function, but this is a much nicer approach! Thx Joan.